src/math.ts

```diff
@@ -10,7 +9,7 @@ const SECONDS_PER_LIQUIDITY_DENOMINATOR = new BN(10).pow(
   new BN(SECONDS_PER_LIQUIDITY_DECIMAL)
 );
 export const POINTS_DENOMINATOR = new BN(10).pow(new BN(POINTS_DECIMAL));
-const SECONDS_INSIDE_DENOMINATOR = POINTS_PER_SECOND;
+const SECONDS_INSIDE_DENOMINATOR = new BN(100);
```

Comment:
We have no domain limits. We can set it to higher value like `10^6`, Maybe we should increase it abit to maintain higher accuracy